ion_hash_tests.ion

```diff
@@ -838,20 +838,6 @@ sexpCount128::{
 
 // structs
 { ion:{}, expect:{ identity:(update::(0x0b) update::(0xd0) update::(0x0e) digest::(0x0b 0xd0 0x0e)) } }
-
-/* FAILING struct tests
-{
-  ion:{$0:1},              // SID0
-  expect:{ identity:(
-    update::(0x0b) update::(0xd0)
-        update::(0x0b) update::(0x71) update::(0x0e)
-        update::(0x0b) update::(0x20) update::(0x01) update::(0x0e)
-        digest::(0x0b 0x71 0x0e 0x0b 0x20 0x01 0x0e)
-      update::(0x0c 0x0b 0x71 0x0c 0x0e 0x0c 0x0b 0x20 0x01 0x0c 0x0e)
-    update::(0x0e)
-    digest::(0x0b 0xd0 0x0c 0x0b 0x71 0x0c 0x0e 0x0c 0x0b 0x20 0x01 0x0c 0x0e 0x0e)
-  ) }
-}
```

Comment:
So we moved the position of the test, I thought they were deleted :). 
Could we add the reason for the test failing as a comment would be great.

Reply:
Added comments.